lib/push-jenkins-update.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const githubClient = require('./github-client')
+
+const jenkinsToGhStatusMap = {
+  'SUCCESS': {
+    state: 'success',
+    message: 'all tests passed'
+  },
+  'FAILURE': {
+    state: 'failure',
+    message: 'build failure'
+  },
+  'ABORTED': {
+    state: 'error',
+    message: 'build aborted'
+  },
+  'UNSTABLE': {
+    state: 'error',
+    message: 'build unstable'
```

Comment:
I've found these statuses by looking at the [build trend](https://ci.nodejs.org/job/node-test-pull-request/buildTimeTrend), this was marked `UNSTABLE`: https://ci.nodejs.org/job/node-test-binary-windows/1866/